documentation/docs/faq.md

```diff
@@ -10,4 +10,27 @@ keywords:
 - IOTA Identity
 ---
 
-TODO: Add list of frequently asked questions and there answers
\ No newline at end of file
+This page contains frequently asked questions regarding the Identity Library and Self Sovereign Identity in general.
+
+### What programming languages are supported by the identity library?
+Please see the "Programming Languages" section, to find out which languages are available.
+### Do I need to have IOTA tokens to start building with identity?
+At the moment you don't need IOTA tokens to create and manage identities, although we are looking at opportunities to utilize the token in the future. 
```

Comment:
Should we mention that we're "working on" rather than "looking at" opportunities? Or maybe "exploring"?

```suggestion
### Do I need to have IOTA tokens to start building with IOTA Identity?
At the moment you don't need IOTA tokens to create and manage identities, although we are exploring opportunities to utilize the token in the future. 
```